.gitignore

```diff
@@ -33,6 +33,7 @@ npm-debug.log
 yarn-error.log
```

Comment:
Please also remove the gitignore entry for `**/ormconfig.js`, sorry I missed this one haha